assets/app/scripts/filters/resources.js

```diff
@@ -781,6 +781,12 @@ angular.module('openshiftConsole')
   })
   .filter('humanizeResourceType', function() {
     return function(resourceType) {
+      if (!resourceType) {
+        return resourceType;
+      }
+
+      // Handle kinds like "ReplicationController" as well
+      resourceType = resourceType.toLowerCase();
```

Comment:
although we are explicitly humanizing differently here when we want the singular...